data/src/commonMain/kotlin/com/wire/kalium/logic/data/mls/CipherSuite.kt

```diff
@@ -77,13 +77,6 @@ sealed class CipherSuite(open val tag: Int) {
         }
     }
 
-    data object MLS_128_X25519KYBER768DRAFT00_AES128GCM_SHA256_ED25519 :
-        CipherSuite(61489) {
-        override fun toString(): String {
-            return "MLS_128_X25519KYBER768DRAFT00_AES128GCM_SHA256_ED25519"
-        }
-    }
```

Comment:
is this CipherSuite removed completely from CC ?

Reply:
yes